### PR TITLE
feat: product categories page

### PR DESCRIPTION
**Pull request by riqwan**
**What:**
- an empty category index page for product categories 
- create category flow with fade modal

Why:

Product categories feature UI completeness.

How:

- Sidebar with new menu item for categories
- Category Route
- Category Index Page in empty state
- BodyCard changed to accept flexible height & border

**Visuals:**
Screenshot 2023-02-08 at 18 29 20

---

RESOLVES CORE-1077
RESOLVES CORE-1078
RESOLVES CORE-1200
RESOLVES CORE-1211

### CODE REVIEW

**Comment by josetr** [comment]
I prefer a nested menu, similar to how Shopify does it.

**Comment by olivermrbl** [comment]
@riqwan How does this one differ from #903 ? :)

File: src/pages/a.jsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ import Pricing from "../domain/pricing"
 import ProductsRoute from "../domain/products"
 import ProductCategories from "../domain/product-categories"
 import PublishableApiKeys from "../domain/publishable-api-keys"
+import ProductCategories from "../domain/product-categories"
```
Comment (olivermrbl) [inline src/pages/a.jsx:22]: todo: Remove duplicate import

**Comment by riqwan** [comment]
closing in favor of https://github.com/medusajs/medusa/pull/3399

**Pull request closed without merging**